com.unity.visualeffectgraph/Documentation~/VisualEffectGraphWindow.md

```diff
@@ -55,11 +56,13 @@ To resize this panel, click on any edge or corner and drag. To reposition this p
 For more information, see [Blackboard](Blackboard.md).
 
 <a name="TargetGameObject"></a>
-### Target Visual Effect GameObject
+### VFX Control
+
+The VFX Control panel displays the controls for the GameObject it is currently attached to. It enables you to:
 
-The **Target Visual Effect GameObject** panel allows you to attach the currently opened Visual Effect Graph to a GameObject. It enables you to:
 * Control playback options
 * Trigger Events
+* Debug Modes
```

Comment:
I forgot to add a verb to this and current saying does not make sense. We could say: 'Use Debug Modes' instead just 'Debug Modes'

Reply:
Fixed